Add update note for available apps in Intune (#5007)
MicrosoftDocs/memdocs: intune/intune-service/apps/macos-unmanaged-pkg.md

1. For the specific app, select either the **Required** or **Available** assignment type.
2. Select **Next** to display the **Review + create** page.

> [!NOTE]

> > If a new version of available app is uploaded to Intune, end user needs to click the install or re-install button on Company Portal to update the app on device.

Reviewer: Thanks for the contribution! The information is accurate. Applying MS writing style guidelines:

```suggestion
> [!NOTE]
> When you upload a new version of an available app to Intune, users must select **Install** or **Reinstall** in Company Portal to update the app on their device.
```

Changes:
- "end user needs to click" → "users must select" (MS style: use "select" for UI actions, avoid "end user")
- Bold UI elements: **Install**, **Reinstall**
- "re-install" → "Reinstall" (no hyphen)
- "on Company Portal" → "in Company Portal" (correct preposition)
- "on device" → "on their device" (added article)
- "available app" → "an available app" (added article)
